src/router/cloudRouter.ts

```diff
@@ -50,16 +51,22 @@ const signInRoute = createRoute({
 	getParentRoute: () => authLayout,
 	path: '/',
 	component: SignIn,
+	beforeLoad: ({ context, location }) => {
+		if (context.authentication.user) {
+			const search: Record<string, string> = location?.search;
+			throw redirect({ to: search?.redirect?.startsWith('/') ? search.redirect : '/orgs' });
+		}
```

Comment:
I only added this check to the sign-in page. I figure someone might legitimately end up at the sign-up page or forgot-password. But if you're on the sign-in page, and you're already signed in, we should redirect you.